### PR TITLE
Atb view all tags

### PR DESCRIPTION
**Pull request by andy-birt**
# Description

This feature includes the ability to view all tags. When a user navigates to the tags section from the navigation bar they will be presented with a list of all available tags. 

## Type of change

[View All Tags #13](https://trello.com/c/emPM8dTz/13-13-view-all-tags)

- [x] New feature (non-breaking change which adds functionality)

# Testing Instructions

1. `git add` and `git commit` all changes in your current branch.
2. `git checkout main`
3. `git fetch -a`
4. `git checkout atb-view-all-tags`
5. Navigate to the root directory for this repo `(directory-to)/tabloidmvc-the-daily-debuggers`
6. `start TabloidMVC.sln`
7. Make sure your db is created with seed data (Skip to 9 if this is already done)
8. Run [01_TabloidMVC_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidmvc-the-daily-debuggers/blob/main/SQL/01_TabloidMVC_Create_DB.sql) and [02_TabloidMVC_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidmvc-the-daily-debuggers/blob/main/SQL/02_TabloidMVC_Seed_Data.sql) queries in Visual Studio
9. Start the server in Debug Mode `F5`
10. Log in as admin `admin@example.com` (if not already logged in as admin)
11. Navigate to the tags link from the Navigation menu
12. You should now see a list of available tags

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by Tommy-Spurlock** [comment]
Looks Good, I Hate it! I would pass it however it needs more cowbell 🔔

**Comment by andy-birt** [comment]
🐮 🔔

File: TabloidMVC/Views/Tags/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+@foreach (var item in Model) {
+        <tr>
+            <td>
+                @Html.DisplayFor(modelItem => item.Id)
+            </td>
+            <td>
+                @Html.DisplayFor(modelItem => item.Name)
+            </td>
+            <td>
+                @Html.ActionLink("Edit", "Edit", new { /* id=item.PrimaryKey */ }) |
```
Comment (Tommy-Spurlock) [inline TabloidMVC/Views/Tags/Index.cshtml:34]: Would be helpful to leave a note somewhere as a reminder to uncomment the links once corresponding actions have been created.

**Review by Tommy-Spurlock — CHANGES_REQUESTED** [review]
Looks good but still needs one more review. Make sure to leave a note for future Andy (or anyone else who may work on this code) for links that don't currently function.

**Review by cbgriffith — APPROVED** [review]
Tags list as they should.  Great work.

**Pull request merged**